account_asset_management_xls/report/account_asset_report_xls.py

```diff
@@ -657,10 +657,29 @@ def _active_report(self, _p, _xs, data, objects, wb):
                     error_name = asset.name
                     if asset.code:
                         error_name += ' (' + asset.code + ')' or ''
-                    raise orm.except_orm(
-                        _('Data Error'),
-                        _("Depreciation Table error for asset %s !")
-                        % error_name)
+                    if asset.state in ['open']:
+                        cr.execute(
+                            "SELECT line_date "
+                            "FROM account_asset_depreciation_line "
+                            "WHERE asset_id = %s AND type = 'depreciate' "
+                            "AND init_entry=FALSE AND move_check=FALSE "
+                            "AND line_date < %s"
+                            "ORDER BY line_date ASC LIMIT 1",
+                            (data[0], fy.date_start))
+                        res = cr.fetchone()
+                        if res:
```

Comment:
There is indeed a bug, I'll make a fix. 
Functional meaning: a clearer error message when trying to sort a report of a fiscal year with entries in prior years that are not posted (we cannot trust the table since we may e.g. have early removals or other changes.